### ISSUE TITLE
Constant reloads

### ISSUE THREAD

**Issue by ohhai-kthxbai** [TEMtheLEM/araa-search#6]
Hello

Thanks for maintaining this, it looks really useful.
Unfortunately I can't get it to work. 
the docker runs fine and I can access the search page but it's constantly reloading, to the point of being useless.
I've tried on multiple browsers.

I appreciate this could be upstream.

Any help appreciated!

**Comment by Extravi** [comment]
are you running it using http and not https and do you have cookies blocked or allowed?

**Comment by TEMtheLEM** [comment]
Just tested it, and yes the page gets stuck in a reload cycle when cookies are completely blocked.

Likely has something to do with the if statement in `/static/uxlang.js` at line 67;
```js
if (!document.cookie.includes("ux_lang")) {
    setLanguageCookie();
}
```
where `setLanguageCookie()` tries to set a cookie, reloading the page after.

**Comment by TEMtheLEM** [comment]
Pushed a fix to docker hub.

Tell me if the issue persists or not.

**Comment by ohhai-kthxbai** [comment]
Thank you for the fast response!

Unfortunately it still has the same issue. I've tried on Firefox, explicitly allowing cookies, also on Chrome (no extensions installed). Chrome is slower to reload but both are unusable.

For DOMAIN I've used the IP assigned to the docker (IPVLAN), could that be breaking it?

**Comment by Extravi** [comment]
it could be an issue running in http

**Comment by Extravi** [comment]
and not https

**Comment by TEMtheLEM** [comment]
1) Just a sanity check; you pulled the `latest` tag again and are still getting this issue?

2) Try setting `DOMAIN` to some public instance as a test. It's only used to generate an `opensearch.xml` file, nothing else needs it.

**Comment by TEMtheLEM** [comment]
Also might be good to know the platform your instance is running on (ie arm/x86, 64/32 bit).

I don't think it matters too much, but there's always a slim chance that there's a deeper issue.

**Comment by ohhai-kthxbai** [comment]
> Just a sanity check; you pulled the latest tag again and are still getting this issue?

Yes, it pulled a new version when you first updated. Lastest tag added and is the same.

> Try setting DOMAIN to some public instance as a test. It's only used to generate an opensearch.xml file, nothing else needs it.

Ok, tried this (set to tailsx.com), no change, still constantly reloads.
I'm running it on Unraid (x86). Let me know what other logs/config I can share to help debug.

**Comment by Nekotje** [comment]
I also have the problem of reloading. When i check the log it shows the 0.0.0.0:8000 so its bound to every connection not sure that matters?
